### PR TITLE
add "sepal-ui" to conda-forge

### PR DESCRIPTION
**Pull request by 12rambau**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by 12rambau** [comment]
@conda-forge-admin, please ping team

**Comment by 12rambau** [comment]
The build failed because of one of my dependencies: planet. 

Is it because it's a pre-release?

**Comment by dopplershift** [comment]
The latest version in the [planet feedstock](https://github.com/conda-forge/planet-feedstock) is 1.5.2, so there's no package meeting the version you're requesting available in conda-forge. We definitely would not want to update to the pre-release on the main conda-forge channel.

**Comment by 12rambau** [comment]
They have forced their users (me included) to upgrade to v2 6 month ago, it's so different that it's impossible to be compatible with both (and way too much work). I guess I'll be forced to wait for the release from their side

**Comment by BastianZim** [comment]
You could ask the maintainer of the feedstock to also publish the pre-release version or make a PR yourself. Some more info is here: https://conda-forge.org/docs/maintainer/knowledge_base.html#pre-release-builds

File: recipes/sepal-ui/meta.yaml
Comment (12rambau) [inline recipes/sepal-ui/meta.yaml:49]:
```suggestion
    - planet >=2.0
```

File: recipes/sepal-ui/meta.yaml
Comment (12rambau) [inline recipes/sepal-ui/meta.yaml:49]:
```suggestion
    - planet >=2.0a6
```

**Comment by 12rambau** [comment]
@BastianZim, the maintainer of the planet feedstock created a [2.0a6 pre-release](https://github.com/conda-forge/planet-feedstock/pull/32), how can I add it to my recipe ? (I tried setting the exact version number but it didn't worked). Would you mind pointing me to the right direction ?

**Comment by BastianZim** [comment]
That’s because they are on a different channel, these docs should help: https://conda-forge.org/docs/maintainer/knowledge_base.html#creating-a-pre-release-build

In short, create a conda_build_config.yaml in your recipe folder and add the following:

channel_sources:
  - conda-forge/label/planet_dev,conda-forge

File: recipes/sepal-ui/meta.yaml
Comment (BastianZim) [inline recipes/sepal-ui/meta.yaml:77]:
```suggestion
    - LICENSE.txt
    - sepal_ui/templates/map_app/LICENSE
    - sepal_ui/templates/panel_app/LICENSE
```

**Review by BastianZim — CHANGES_REQUESTED** [review]
Just thing and this is good to go.

**Pull request merged**
